### PR TITLE
Updated Everfund URL + added UTM's

### PR DESCRIPTION
**Pull request by Burnsy**
- Updated Everfund url to us website
- Added UTM's so said websites will be able to see where the users are coming from on there analytics

### CODE REVIEW

File: web/src/components/FundingShoutout/FundingShoutout.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ const FundingShoutout = () => {
         <div className="mx-auto mt-6 grid grid-cols-1 gap-0.5 overflow-hidden rounded-xl md:grid-cols-3 lg:mt-8 lg:max-w-6xl">
           {companies.map((company, i) => (
             <a
-              href={company.link}
+              href={`${company.link}?utm_source=redwoodjs&utm_medium=logo+area}
```
Comment (cannikin) [inline web/src/components/FundingShoutout/FundingShoutout.tsx:60]: No closing quote on this string!

**Review by cannikin — CHANGES_REQUESTED** [review]

File: web/src/components/FundingShoutout/FundingShoutout.tsx (hunk 1 above)
Comment (cannikin) [reply web/src/components/FundingShoutout/FundingShoutout.tsx:60]: Also: I don't know that we should hardcode these utm vars here for all links...other sites may have their own custom vars they want to use! They can do that by updating their own link to include anything they want.

Comment (Burnsy) [reply web/src/components/FundingShoutout/FundingShoutout.tsx]: Rolled back the hardcoded UTM's for just ones on everfund :)

**Comment by Burnsy** [comment]
LGTM, Fixed Everfund url to include UTM. Removed hardcoded

**Pull request merged**